dataset_config/1000_genomes/ui.json

```diff
@@ -89,5 +89,5 @@
     }
   ],
   "enable_search_values": true,
-  "show_viz_toggle": false
+  "show_viz_toggle": true
```

Comment:
Revert.

(For local runs, I create a separate 1000_genomes_viz directory)

Reply:
Done.